packages/client/hmi-client/src/components/models/tera-stratified-value-matrix.vue

```diff
@@ -118,6 +155,28 @@ function getMatrixValue(variableName: string) {
 	return variableName;
 }
 
+async function getMatrixValue(variableName: string, shouldEvaluate: boolean) {
+	const expressionBase = getMatrixExpression(variableName);
+
+	if (shouldEvaluate) {
+		const expressionEval = (await pythonInstance.evaluateExpression(
+			expressionBase,
+			parametersValueList.value
+		)) as string;
+		return (
+			(await pythonInstance.parseExpression(expressionEval)) as {
```

Comment:
Why not update create a `parseExpressionToMathML()` that return a `string` directly, so you don't have to do this weird typing `( .... as { mathml: string; }).mathml;`

Reply:
this has been fixed by updating the `parseExpression` type